tests/lib/bash/test_header

```diff
@@ -549,6 +555,30 @@ poll_suite_stopped() {
     poll test '!' -e "${SUITE_RUN_DIR}/.service/contact"
 }
 
+get_suite_uuid() {
+    sed -n 's|CYLC_SUITE_UUID=\(.*\)|\1|p' "${SUITE_RUN_DIR}/.service/contact" 2>/dev/null
+}
+
+poll_suite_restart() {
```

Comment:
`poll_suite_stopped` can't reliably detect suite restart as the period in which the contact file is absent might be less than the polling period.